GameServerLib/Game.cs

```diff
@@ -141,57 +167,61 @@ public void NetLoop()
             _lastMapDurationWatch.Start();
             while (!SetToExit)
             {
-                while (_server.Service(0, out var enetEvent) > 0)
+                lock (this)
                 {
```

Comment:
Is this lock necessary? We don't use much threading. If so, please create new locking object instead of locking "this".

Reply:
I’ll make a new lock.
It’s necessary because GameServerLib is weird and locks the thread it’s running on with a while loop. Means the entire thing has to run in a separate thread in a UI app.
[Not yet resolved]

Reply:
Resolved